### PR TITLE
August 2017 meeting

### PR DESCRIPTION
**Pull request by tobyhodges**
The minutes from yesterdays meeting. If you prefer, I can re-submit after we've migrated to the new carpentries repo?

### CODE REVIEW

**Comment by ChristinaLK** [comment]
Let's merge + we can start fresh in September in the new repo.

File: subcommittees/mentoring/minutes/minutes-2017-08-28.md [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+  - The Software and Data Carpentry re-organization is moving along. See the latest blog post announcement here: https://software-carpentry.org/blog/2017/08/mergermotions.html  Relevant information for this community: 
+    - In the future, sub-committees will have a staff point person, so after the merger, someone from the new Carpentries organization will be helping with the mentoring committee
+    - The  mentoring committee is something that applies across carpentries, so we will be helping move out of the swcarpentry/board repo into a carpentries/mentoring repo soon
+  - More discussion is happening about CarpentryCon, people in this committee might be interested in checking it out.
+    - http://pad.software-carpentry.org/2018carpentrycontaskforce
+
+###### First Meeting
+
+- MK: now the CarpentryCon committee knows more what they want - more of an unconference, keynote, interactive sessions - two days with an extra day for community-organized events - but two days would be the core - more venue requirements set out by group
+- MK: links to carpentrycon documents: https://drive.google.com/open?id=0B70o76-4Jmd0NzRteURZRWRhZGc
```
Comment (mkuzak) [inline subcommittees/mentoring/minutes/minutes-2017-08-28.md]: this link does not work and I cannot get the shareable link to this drive, I'd remove it, at least until I know why it does not work

File: subcommittees/mentoring/minutes/minutes-2017-08-28.md [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+
+#### *Walk-ons: please add additional topics below
+
+- MK: we should contribute to the __new carpentries repo__. I'd like to start to do this. anyone else want to help?
+  - TH: +1
+  - CK: +1
+  - JH: +1
+
+- PA: __new community for Latin America__. Translating lessons to Spanish, hosting spanish language discussion sessions etc
+  - MK: Raniere is already hosting a few sessions in spanish, I think
```
Comment (mkuzak) [inline subcommittees/mentoring/minutes/minutes-2017-08-28.md]: I meant Rayna (not Raniere), but @rgaiacs may have been hosting those too :)

Comment (raynamharris) [reply subcommittees/mentoring/minutes/minutes-2017-08-28.md]: Rayna and Sue McClatchy are hosting demo sessions in Spanish.

Comment (raynamharris) [reply subcommittees/mentoring/minutes/minutes-2017-08-28.md]: Rayna and Sue McClatchy are hosting demo sessions in Spanish.

File: subcommittees/mentoring/minutes/minutes-2017-08-28.md (hunk 1 above)
Comment (tobyhodges) [reply subcommittees/mentoring/minutes/minutes-2017-08-28.md]: I think this is the right link - https://drive.google.com/drive/u/1/folders/0B2Mb22lbj2tjMFBiVE5oQ3ctZHc

but I can't access without permission (which Ive now requested), so I propose that I instead encourage people in the minutes to attend the ext CarpentryCon call on 7th Sep instead.

**Comment by mkuzak** [comment]
merged and migrated to new carpentries/mentoring

**Pull request merged**